source/dem/dem.cc

```diff
@@ -321,6 +323,12 @@ DEMSolver<dim>::load_balance()
           triangulation, solids[i_solid]->get_solid_triangulation());
     }
 
+  if (parameters.boundary_conditions.BC_type ==
+      Parameters::Lagrangian::BCDEM::BoundaryType::periodic)
+    {
+      periodic_boundaries_object.map_periodic_cells(triangulation);
+    }
+
```

Comment:
Here this will have to be replaced with a loop over boundaryconditions no? if that's the case,I would put this small verification into some sort of utility.

Reply:
So far parameters are stored in vectors to allow multiple periodic walls, but only parameters in [0] are used, which makes only one periodic wall available. I should modify the PR to implement it